recipes/seispy/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "seispy" %}
+{% set version = "1.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/python-{{ name }}/python-{{ name }}-{{ version }}.tar.gz
+  sha256: bdd1bbcd2de70ab232eada27d86ed96a292191c9f0c841c2141c0029735e4649
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - gen_rayp_lib=seispy.psrayp:gen_rayp_lib
+    - prf=seispy.rf:prf
+    - setpar=seispy.rf:setpar
+    - rf2depth=seispy.rf2depth_makedata:rf2depth
+    - plotrt=seispy.plotRT:main
+    - plotr=seispy.plotR:main
+    - updatecatalog=seispy.updatecatalog:main
+    - ndk2dat=seispy.updatecatalog:ndk2dat
+    - ccp_profile=seispy.scripts:ccp_profile
+    - hk=seispy.hk:hk
+    - pickrf=seispy.pickui:main
+    - rfani=seispy.scripts:rfani
+    - ccp3d=seispy.scripts:ccp3d
+    - get_pierce_points=seispy.scripts:get_pierce_points
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib >=3.2.0
+    - netcdf4 >=1.5.2
+    - numpy >=1.19.0
+    - obspy >=1.2.0
+    - pandas >=1.0.0
+    - pyerf >=1.0.1
+    - pyqt >=5.12.0
+    - python >=3.6
+    - scipy >=1.1.0
+
+test:
+  imports:
+    - seispy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/python-seispy/
+  license: GPL-2.0-or-later
```

It's actually GPL v3
```suggestion
  license: GPL-3.0-or-later
```